### PR TITLE
cleaning up test env vars

### PR DESCRIPTION
**Pull request by brettsam**
PR #9325 introduced a test that does not reset environment variables when they get set. This can lead to subsequent tests failing because the env vars haven't been cleaned up.

### CODE REVIEW

File: test/WebJobs.Script.Tests/Extensions/ScriptJwtBearerExtensionsTests.cs [hunk 1]
```diff
@@ -24,22 +23,21 @@ public void CreateTokenValidationParameters_HasExpectedAudience(bool isPlacehold
             var podName = "RandomPodName";
             var containerName = "RandomContainerName";
             var siteName = "RandomSiteName";
-            ScriptSettingsManager.Instance.SetSetting(AzureWebsiteName, siteName);
```
Comment (brettsam) [inline test/WebJobs.Script.Tests/Extensions/ScriptJwtBearerExtensionsTests.cs:27]: behind-the-scenes, this is setting env vars that are not being cleaned up.

File: test/WebJobs.Script.Tests/Extensions/ScriptJwtBearerExtensionsTests.cs [hunk 2]
```diff
@@ -54,17 +52,13 @@ public void CreateTokenValidationParameters_HasExpectedAudience(bool isPlacehold
...
-                    ScriptSettingsManager.Instance.GetSetting(ContainerName)
-                };
+                expectedWithContainerName = new string[] { containerName };
                 testData[AzureWebsiteInstanceId] = string.Empty;
                 testData[ContainerName] = containerName;
             }
 
             testData[ContainerEncryptionKey] = Convert.ToBase64String(TestHelpers.GenerateKeyBytes());
-            using (new TestScopedEnvironmentVariable(testData))
+            using (new TestScopedSettings(ScriptSettingsManager.Instance, testData))
```
Comment (brettsam) [inline test/WebJobs.Script.Tests/Extensions/ScriptJwtBearerExtensionsTests.cs:61]: Wrapping it all in here ensures that it'll bet cleaned up on disposal.

**Comment by brettsam** [comment]
@yojagad / @Tonewall -- can you double-check this looks okay? I basically just set up the test var dictionary then apply it wrapped in the `using` rather than doing it first...

**Review by yojagad — APPROVED** [review]
:shipit:

**Review by liliankasem — APPROVED** [review]

**Pull request merged**